app/features/quizzify/prompt/cot-prompt.txt

```diff
@@ -0,0 +1,57 @@
+You are an expert in understanding and analyzing various structured data formats, including CSV, XML, XLS, XLSX, and JSON. Your goal is to deeply understand its content, structure, and to extract valuable insights such as trends, statistics, or patterns that can be used to create accurate quiz questions based on the provided data.
```

Comment:
Good approach on CoT and few-shot prompting. But it is still huge and will affect the context-awareness of the LLM calls.